pkgs/modules/bun/default.nix

```diff
@@ -6,17 +6,30 @@ let
   extensions = [ ".json" ".js" ".jsx" ".ts" ".tsx" ];
 
   community-version = lib.versions.majorMinor bun.version;
+
+  package-json-runner = pkgs.writeScript "bun-runner" ''
+    #!${pkgs.bash}/bin/bash
+    if start=$(${pkgs.jq}/bin/jq -r '.scripts.start? // empty' package.json 2>/dev/null); [[ -n $start ]]; then
+      echo "+ bun run start";
+      ${bun}/bin/bun run start;
+    elif module=$(${pkgs.jq}/bin/jq -r 'if .type == "module" then .module // empty else empty end' package.json 2>/dev/null); [[ -n $module ]]; then
+      echo "+ bun $module";
+      ${bun}/bin/bun $module;
+    elif main=$(${pkgs.jq}/bin/jq -r '.main // empty' package.json 2>/dev/null); [[ -n $main ]]; then
+      echo "+ bun $main";
+      ${bun}/bin/bun $main;
+    elif [[ -n $file ]]; then
+      echo "+ bun $file";
+      ${bun}/bin/bun $file;
+    fi
+  '';
 in
 
 {
   id = "bun-${community-version}";
   name = "Bun Tools";
 
   imports = [
-    (import ../run-package-json {
```

Comment:
Are you able to delete this directory now too?

Reply:
i still plan on fixing `run-package-json`, i just want to do that in a different pr. the script in this pr is the v3 runner script